sql/core/src/test/resources/sql-tests/inputs/string-functions.sql

```diff
@@ -14,4 +14,4 @@ select replace('abc', 'b', '123');
 select replace('abc', 'b');
 
 -- uuid
-select length(uuid()), (uuid() <> uuid());
```

Comment:
I think what @ueshin means is we should set `deterministic = false` for `Uuid` expression.